.github/workflows/causal_lm_cpp.yml

```
@@ -212,6 +212,29 @@ jobs:
           source ./ov/setupvars.sh
           convert_tokenizer ./notus-7b-v1/pytorch/dldt/FP16/ --output ./notus-7b-v1/pytorch/dldt/FP16/ --with-detokenizer --trust-remote-code
           timeout 50s ./build/beam_search_causal_lm ./notus-7b-v1/pytorch/dldt/FP16/ 69 > ./pred.txt
-
-
-          
+  cpp-beam_search_causal_lm-Phi-1_5:
```

please remove mentioning decoding algorithm and leave only model name

```suggestion
  cpp-Phi-1_5:
```